lib/translator/logsense.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LIB_TRANSLATOR_LOGSENSE_H
+#define LIB_TRANSLATOR_LOGSENSE_H
+
+#include "absl/strings/string_view.h"
+#include "absl/types/span.h"
+#include "common.h"
+#include "lib/scsi_defs.h"
+#include "third_party/spdk_defs/nvme_defs.h"
+
+namespace translator {
+void translate(const scsi_defs::LogSenseCommand &cmd, absl::Span<uint8_t>);
```

Comment:
This PR doesn't seem to follow the pattern other PRs have been using. That is, the header exposes a "CommandToNvme" and "CommandToScsi" function. All other functions are part of an anonymous namespace